.github/workflows/push.yml

```diff
@@ -164,4 +164,28 @@ jobs:
         user_email: '${{env.email}}'
         user_name: '${{github.actor}}'
         commit_message: 'Talawa API docs updated'
-      
+        
+# You can find the deployment instructions in the scripts/cloud-api-demo/README.md file
+  Deploy-Workflow:
+    name: Deploying Application to Cloud VPS
+    runs-on: ubuntu-latest
+    steps:
+      - name: Execute remote ssh commands 
+        uses: appleboy/ssh-action@v1.0.3 
+        env:
+          INPUT_ACCESS_TOKEN_SECRET: ${{ secrets.API_DEMO_ACCESS_TOKEN_SECRET }}
+          INPUT_REFRESH_TOKEN_SECRET: ${{ secrets.API_DEMO_REFRESH_TOKEN_SECRET }}
+          INPUT_RECAPTCHA_SECRET_KEY: ${{ secrets.API_DEMO_RECAPTCHA_SECRET_KEY }}
+          INPUT_MAIL_USERNAME: ${{ secrets.API_DEMO_MAIL_USERNAME }}
+          INPUT_MAIL_PASSWORD: ${{ secrets.API_DEMO_MAIL_PASSWORD }}
+          INPUT_LAST_RESORT_SUPERADMIN_EMAIL: ${{ secrets.API_DEMO_LAST_RESORT_SUPERADMIN_EMAIL }}
+        with:
+          host: ${{ secrets.API_DEMO_HOST }}
+          username: ${{ secrets.API_DEMO_USERNAME }}
+          key: ${{ secrets.API_DEMO_SSH_KEY }}
+          port: ${{ secrets.API_DEMO_PORT }}
```

Comment:
Change to `API_DEMO_SSH_PORT` for consistency. This is what I have configured in the GitHub portal

Reply:
Done